guidelines.md

```diff
@@ -52,6 +52,7 @@ The API responds with `Content-Type` set to `application/json` and JSON content.
 * `401 Unauthorized` - Error caused by usage of invalid access token.
 * `403 Forbidden` - Server error that should be reported to the end user of the client app. Happens for example when the server-side validation fails or when a business-logic check is violated.
 * `500 Internal Server Error` - Unexpected error of the server. In most cases, such an error signifies a bug on our side. We are logging it and immediately notified when such error happens. If anything like this happens, feel free to directly contact us or raise an issue here on Github.
+* `429 Too Many Requests` - Error caused by too many requests sent in a given amount of time for the same `Client Token`, `Access Token`, or endpoint per `Access Token`. [Request limits](guidelines.md#request-limits) have been reached. Wait 15 minutes before trying again. 
```

Comment:
nitpick - move above so that it's sorted. I'd simplify this to state just "Error caused by too many requests sent in a given amount of time, please see [Request limits](guidelines.md#request-limits) for more information."

Reply:
good point, reordered it + shortened description